akka-cluster/src/main/scala/akka/cluster/ClusterDaemon.scala

```diff
@@ -1065,15 +1070,18 @@ private[cluster] class ClusterCoreDaemon(publisher: ActorRef) extends Actor with
     val localSeen = localOverview.seen
 
     val enoughMembers: Boolean = isMinNrOfMembersFulfilled
```

Comment:
one could argue that both are useful, but I'm also leaning towards only counting members in the team because leader actions are all based on that and a network partition across should not halt another team's action